src/Python/requirements.txt

```diff
@@ -2,11 +2,11 @@
 ipython==8.10.0      # to avoid high-severity issue.
 jupyter==1.0.0
 nbconvert==7.0.0
-notebook==6.4.12
+notebook==6.5.2
 numpy==1.22.0
 pytest==7.2.0
 pygments==2.12.0    # to avoid high-severity issue.
-pyzmq==23.2.1       # to workaround: https://github.com/zeromq/pyzmq/issues/1764
+pyzmq==25.0.0       # to workaround: https://github.com/zeromq/pyzmq/issues/1764
```

Comment:
This comment seems obsolete now? Seems like the pin to version 23.x was added [here](https://github.com/microsoft/iqsharp/pull/720) to work around a specific build issue, which sounds like isn't relevant anymore.

Reply:
Yes, you are right. This comment seems to be outdated now. Since this PR is already complete, I will remove it in this PR: https://github.com/microsoft/iqsharp/pull/772